src/dhcp6relay/src/configInterface.cpp

```diff
@@ -135,6 +136,9 @@ void processRelayNotification(std::deque<swss::KeyOpFieldsValuesTuple> &entries,
             if(f == "dhcpv6_option|rfc6939_support" && v == "false") {
                 intf.is_option_79 = false;
             }
+            if(f == "dhcpv6_option|interface_id" && v == "true") { // interface-id is off by default on non-Dual-ToR, unless specified in config db
+                intf.is_interface_id = true;
+            }
```

Comment:
What if config_db changed from true to false? You only allow one direction change?

Same question for 136-138.

Reply:
option79 is true and interface_id is false by default(line 120-121) if not specified in config_db
changes in config_db will require dhcp6relay restart to take effect